gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+## Title
+* Create an intranet marketplace where InnerSource project owners and potential contributors can easily discover one another
+
+## Patlet
+Establish a marketplace by creating a website(s) that lists InnerSource project needs as "gigs". 
```

Comment:
Kind of sounds a bit like [Top Coder](https://www.topcoder.com/) but internal to the company?

Reply:
That's the general idea, but the intended outcomes are so much different.  The Gigs posted are also used as "proof of ability" for the developers accepting them after they are completed.  Another main objective is for developers, not managers, to post Gigs allowing them to work with one another to accomplish just about anything they can think of.  It's a mechanism that flattens the organizational hierarchy, ignores BU affiliations and opens up the whole company wide development community as a resource for anyone to potentially leverage.

Reply:
Fun!

Reply:
Based on comments from @rshanmer : looks good, length wise. Also focuses on solution, which is how it should be. Possible elaborate on the content of the 2nd sentence.

Reply:
from @NewMexicoKid : Would it be helpful to introduce a length limit of patlets?
@rshanmer : Maybe limit to two sentences, instead of going for a #characters limit.
all: two sentences sounds like a reasonable choice.